### DISCUSSION TITLE
相册详情页面可能存在的问题

### DISCUSSION THREAD

**Discussion by zielorem** [MaLuns/hexo-theme-async#58] [Q&A 疑难杂症] [answered]
使用标签外挂galleryGroup设定相册详情页面时，若引用本地gallery目录下的图片，如
`{% galleryItem nodz_cover.jpeg cover.png %}`
理论上nodz_cover.jpeg为缩略图，cover.png为点击该图片后的大图，但在实际点击该图片后大图无法加载。
<img width="620" alt="截屏2023-01-28 04 17 54" src="https://user-images.githubusercontent.com/108322419/215189545-a4ae9fac-1500-48e4-b720-54b995a4c9cb.png">
同时terminal中一直都只传递了前一张图，后一张图可能因此才无法找到。
<img width="785" alt="截屏2023-01-28 04 22 29" src="https://user-images.githubusercontent.com/108322419/215189833-ff402b92-da7c-4550-b888-6aa9ec8cda7f.png">
另外本地图片通过Markdown格式直接引用同样也不能显示（lazyload已关闭）
有空的话还请作者大大解决一下该问题(

**Comment by MaLuns** [comment, marked as answer]
预览大图路径生成对吗？使用 galleryItem 目前还不支持相对路径，可以先补全路径。
